src/firefly/js/core/background/JobInfo.jsx

```diff
@@ -79,12 +81,13 @@ export function JobInfo({jobId, ...props}) {
 }
 
 export function UwsJobInfo({jobInfo, sx, isOpen=false}) {
-    const {results, parameters, errorSummary, meta, jobInfo:aux, ...rest} = jobInfo;
+    const {results, parameters, errorSummary, jobInfo:aux} = jobInfo;
     const hrefs = results?.map((r) => r.href);
     const hasMoreSection = hrefs || parameters || errorSummary || aux;
     return (
         <Stack spacing={.5} p={1} sx={sx}>
```

Comment:
```suggestion
        <Stack spacing={1} p={1} sx={sx}>
```
Looks tight, JobInfoDetails, TAPDetails, etc. already have their children with 0.5 vertical space (at least visually). To make these visual groups separate, consider using higher vertical spacing.